mount.cc

```diff
@@ -27,11 +27,19 @@ class MountWorker : public Nan::AsyncWorker {
   // This function is executed in another thread at some point after it has been
   // scheduled. IT MUST NOT USE ANY V8 FUNCTIONALITY.
   void Execute() {
-    int ret = mount(mounty->devFile.c_str(),
-            mounty->target.c_str(),
-            mounty->fsType.c_str(),
-            mounty->flags,
-            mounty->data.c_str());
+    #ifdef __linux__
+      int ret = mount(mounty->devFile.c_str(),
+              mounty->target.c_str(),
+              mounty->fsType.c_str(),
+              mounty->flags,
+              mounty->data.c_str());
+    #elif __APPLE__
+      int ret = mount(
+              mounty->fsType.c_str(),
+              mounty->target.c_str(),
+              mounty->flags,
+              &mounty->data);
```

Comment:
Hm. That's odd. It should be a pointer... Weird.
https://www.gnu.org/software/libc/manual/html_node/Mount_002dUnmount_002dRemount.html